packages/opentelemetry-core/src/baggage/constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const KEY_PAIR_SEPARATOR = '=';
+export const PROPERTIES_SEPARATOR = ';';
+export const ITEMS_SEPARATOR = ',';
+
+// Name of the http header used to propagate the baggage
+export const BAGGAGE_HEADER = 'baggage';
```

Comment:
Why move these constants out of the propagator when they aren't used or needed by anything else outside the propagator?

Reply:
Well there are used by the collector exported for parsing the headers from env vars (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specifying-headers-via-environment-variables)

Reply:
Example: https://github.com/open-telemetry/opentelemetry-js/pull/2117/files#diff-e79e0fea33db7d679a3e0cd9bc2fa4e6ed82bcab240d68fce3fe60c2f2f94235R43